flocker/acceptance/test_api.py

```diff
@@ -85,8 +85,7 @@ def _create_container(self, cluster):
 
         d = cluster.create_container(data)
 
-        def check_result(result):
-            cluster, response = result
+        def check_result(response):
```

Comment:
Update `_create_container` docstring `:return:`. It only returns the container dictionary now.